test/e2e/watchcache/watchcache_enabled_test.go

```diff
@@ -62,17 +62,17 @@ func TestWatchCacheEnabledForCRD(t *testing.T) {
 	t.Cleanup(cancel)
 	org := framework.NewOrganizationFixture(t, server)
 	cluster := framework.NewWorkspaceFixture(t, server, org)
```

Comment:
schedule to root shard via option

Reply:
i am sorry, i don't follow, can you elaborate?

Reply:
`framework.NewWorkspaceFixture(t, server, org, framework.WithShardConstraints(tenancyapi.ShardConstraints{Name: "root"})),`

Reply:
fixed